Dockerfile

```diff
@@ -46,9 +46,9 @@ RUN apt-get -qq update \
   && rm /tmp/sym-collector.zip \
   && chmod +x /usr/local/bin/SymbolCollector.Console
 
-RUN curl -fsSL https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.7.1-stable.tar.xz -o /tmp/flutter_linux_3.7.1-stable.tar.xz \
-  && tar xf /tmp/flutter_linux_3.7.1-stable.tar.xz \
-  && rm /tmp/flutter_linux_3.7.1-stable.tar.xz
+RUN curl -fsSL https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.7.1-stable.tar.xz -o /opt/flutter_linux_3.7.1-stable.tar.xz \
+  && tar xf /opt/flutter_linux_3.7.1-stable.tar.xz -C /opt \
+  && rm /opt/flutter_linux_3.7.1-stable.tar.xz
```

Comment:
it doesn't matter but the tgz should sit in `/tmp` but the unpack should be in `/opt`